src/Compiler/Backend/GLSL/GLSLConverter.cpp

```diff
@@ -1008,6 +1012,50 @@ void GLSLConverter::ConvertIntrinsicCallImageAtomic(CallExpr* ast)
     }
 }
 
+void GLSLConverter::ConvertIntrinsicCallImageStore(CallExpr* ast)
+{
+    /* Cast input value argument to 4 component vector */
+    auto& args = ast->arguments;
+
+    if (args.size() >= 3)
+    {
+        DataType dataType = DataType::Float;
+
+        const auto& arg0Expr = args.front();
+        if (auto arg0ArrayExpr = arg0Expr->As<ArrayExpr>())
```

Comment:
Don't forget brackets, they can appear almost everywhere and must be 'circumvented' similar to the AliasTypeDenoter with `GetAliased`:
```cpp
auto nonBracketExpr = args.front()->FindFirstNotOf(AST::Types::BracketExpr);
```